packages/material-ui-lab/src/ToggleButton/ToggleButton.js

```diff
@@ -66,6 +67,19 @@ export const styles = theme => ({
     alignItems: 'inherit',
     justifyContent: 'inherit',
   },
+  /* Styles applied to the root element if `size="small"`. */
+  sizeSmall: {
+    height: 40,
+    minWidth: 40,
+    fontSize: theme.typography.pxToRem(13),
+  },
+
+  /* Styles applied to the root element if `size="large"`. */
+  sizeLarge: {
+    height: 56,
+    minWidth: 56,
```

Comment:
I believe this needs to be +1px to account for the negative margin (if the buttons are to remain square).